homeassistant/components/netgear/switch.py

```diff
@@ -106,3 +195,67 @@ def async_update_device(self) -> None:
             self._state = None
         else:
             self._state = self._device[self.entity_description.key] == "Allow"
+
+
+class NetgearRouterSwitchEntity(NetgearRouterEntity, SwitchEntity):
+    """Representation of a Netgear router switch."""
+
+    _attr_entity_registry_enabled_default = False
+    entity_description: NetgearSwitchEntityDescription
+
+    def __init__(
+        self,
+        router: NetgearRouter,
+        entity_description: NetgearSwitchEntityDescription,
+    ) -> None:
+        """Initialize a Netgear device."""
+        super().__init__(router)
+        self.entity_description = entity_description
+        self._name = f"{router.device_name} {entity_description.name}"
+        self._unique_id = f"{router.serial_number}-{entity_description.key}"
+
+        self._state = None
```

Comment:
Use `self._attr_is_on` and you don't need to create the `is_on` property
```suggestion
        self._is_on = None
```

Reply:
done, thanks for the hint!